client/cypress/integration/login_spec.js

```diff
@@ -0,0 +1,51 @@
+import axios from 'axios'
+/* global cy */
+
+describe('Login page', () => {
+    
+    let user = {
```

Comment:
Just as an FYI, another handy thing about Cypress are "fixture files" where you'd place this type of static mock data that can be used across multiple tests: https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests.html#Fixture-Files

![image](https://user-images.githubusercontent.com/9777763/98390401-37764300-204d-11eb-9365-b9f61465bec9.png)

Reply:
Good to know!